src/utils/clients/discogs/formatters/album-search-response.formatter.ts

```diff
@@ -0,0 +1,31 @@
+interface AlbumResult {
+  type: string;
+  id: number;
+  master_id?: number;
+  title: string;
+  year: number;
+  thumb: string;
+  cover_image: string;
+}
+
+export function albumSearchFormatter(response?: { data?: { results?: AlbumResult[] } }) {
```

Comment:
... which would simplify this to something like:

```suggestion
export function albumSearchFormatter(response: any): AlbumResult[] {
```

Reply:
yes, I understand. Maybe I can define it as a Json object to avoid the any. Let me check

Reply:
A JSON generic may work initially, but in the end you will need to define actual "types" locally whenever you try to use the object. I changed this to any in 8a2863f so we can review this later if you want to change the typescript check to avoid any directly.